src/extensions/surveys/surveys-utils.tsx

```diff
@@ -604,6 +625,24 @@ export const getDisplayOrderQuestions = (survey: Survey): SurveyQuestion[] => {
     return reverseIfUnshuffled(survey.questions, shuffle(survey.questions))
 }
 
+export const getSurveySeenKey = (survey: Survey): string => {
+    let surveySeenKey = `seenSurvey_${survey.id}`
+    if (survey.current_iteration && survey.current_iteration > 0) {
+        surveySeenKey = `seenSurvey_${survey.id}_${survey.current_iteration}`
+    }
```

Comment:
camelCase and snake_case?  Is this a pattern for tracking metadata after variables?  Reads a bit odd to me but I could just be overthinking it.

Reply:
yeah, its the pattern used for these sorts of keys. original feedback on this variable is in https://github.com/PostHog/posthog-js/pull/1200#discussion_r1618758364